content/ri/using-redisinsight/api-get.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Adding Databases via GET URL
+date: 2021-01-11 00:00:00 +0000
+weight: 16
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
+If you want to automate adding a Redis database without filling the [database form]({{< relref "/ri/using-redisinsight/add-instance.md" >}}), you might want to add database via GET URL.
+
+Below is the documentation for the URL required to add databases.
+
+## Add Redis database
+
+Used to add Redis databases to RedisInsight.
+
+**URL** : `/add/`
+
+**Method** : `GET`
+
+
+### Query Parameters
+
+These are the required query parameters for any type of database.
+
+| Parameter        | Type    | Required/Optional | Description                                                                  |
+|------------------|---------|-------------------|------------------------------------------------------------------------------|
+| `host`           | string  | required          | Hostname of your Redis database.                                             |
+| `port`           | number  | required          | Port of your Redis database.                                                 |
+| `name`           | string  | optional          | A nick name for the Redis database. Any string is valid                      |
+| `username`       | string  | optional          | Username of your Redis database.                                             |
+| `passsword`      | string  | optional          | Password of your Redis database.                                             |
+| `tls`            | boolean | optional          | `"true"` if your Redis database has TLS enabled.                             |
+| `verifyServer`   | boolean | optional          | `"true"` if the server certificate of you Redis database has to be verified. |
+| `masterName`     | string  | optional          | master name of the monitoring database                                       |
+| `masterUsername` | string  | optional          | master username of the monitoring database                                   |
+| `masterPassword` | string  | optional          | master password of the monitoring database                                   |
+| `redirect`       | boolean | optional          | `"true"` if you want to redirect to instance page after successful addition. |
+|                  |         |                   |                                                                              |
+
+{{< note >}}
+If you want to add the TLS certificates for your Redis database, you have to manually fill [database form]({{< relref "/ri/using-redisinsight/add-instance.md" >}}).
+{{< /note >}}
+
+
+#### Examples
```

Comment:
Could we just give examples of the URL text rather than an entire screenshot?

Reply:
I think that should be fine. @GnaneshKunal Can you make the necessary changes, please?

Reply:
Does the pattern of the URL looks fine or can I change the pattern `http://<redisinsight-host>:<redisinsight-port>/add/....`?

![image](https://user-images.githubusercontent.com/20211167/107010634-99fded00-67bc-11eb-9b0f-bb851fae15ab.png)

Reply:
I think we definitely need to differentiate what is a variable and what is not. Maybe we could have two URLs, one using variable names and an example with sample values? That way the user can see both which values they need to supply and what they look like in context. 

http://add/?name=<redisinsight-host>&host=<local-host>&port=<redisdb-prt>
http://add/?name=mydb&host=172.19.02&port=6379